source/includes/_application-api.md

```diff
@@ -0,0 +1,453 @@
+# Your application API
+
+The API is the way the outside world interact with your system, and any server-side application needs one.
+With Booster, you don't need to worry about it: _it is created for you_ and evolves with your code _automatically_.
+
+When you finish deploying your application using the `boost deploy` command, it will print useful information needed to 
+use your API. It will be shown under the "Outputs" section.
+
+```sh
+Outputs:
+<application name>.httpURL = https://<API ID>.execute-api.<region>.amazonaws.com/<environment name>
+<application name>.websocketURL = wss://<API ID>.execute-api.<region>.amazonaws.com/<environment name>
+<application name>.clientID = abcdXjk1234
+```
```

Comment:
Note that this is explaining how it works in AWS. Another provider might not have this code. I'd remove this and say that after the application is deployed, the endpoints will be shown

Reply:
I agree, but let's do it in another PR in the new docs, just to merge this and move the docs there